examples/XX-anomaly-detection.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # 8 anomalous days
</pre><p>Should be 5 I guess</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1518/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>